src/core/InputDropdown/style.ts

```diff
@@ -338,3 +323,49 @@ export const StyledCounter = styled("span", {
   `;
   }}
 `;
+
+interface MinimalDetailsProps extends CommonThemeProps {
+  shouldTruncateMinimalDetails: InputDropdownProps["shouldTruncateMinimalDetails"];
+}
+
+export const MinimalDetails = styled("div")`
+  ${labelFontBodyS}
```

Comment:
This is now the only InputDropdown variant that needs to override the default xs font size to be size s instead (for the minimal variant's label).

The `interface` statement above lets the `labelFontBodyS` variable get the props it needs (@tihuan if I am correctly explaining?)